### ISSUE TITLE
Sub-basin: Add List Table for NHD+ Catchment results

### ISSUE THREAD

**Issue by arottersman** [WikiWatershed/model-my-watershed#2730]
Under the "Catchments" tab in the sub-basin HUC-12  view, add a table displaying the total loads, loading rates, and mean annual concentrations for each catchment of the HUC-12.

<img width="1008" alt="38377615-f1465ba2-38c0-11e8-94ec-7bb85a7be16a" src="https://user-images.githubusercontent.com/7633670/38559155-3f2b00c4-3ca0-11e8-9440-b3d2ca9f34cd.png">

Total Loads should correspond to the `TotalLoadingRates` for each catchment, Loading Rates should be calculated as the `TotalLoadingRates` over the area of the catchment (we'll have to look this up from the `nhdcatchments` table via the catchment's comid). The Mean Annual Concentrations are in `LoadingRateConcentrations`.

[Abstract wireframes here, use for style but not table content](https://app.goabstract.com/projects/76ed1390-dea8-11e7-a27c-0f868efbc56f/branches/master/commits/a49748bccbdf373c6c9c88eae15b2ee1a73d5e71/files/7F4A25A6-1E0D-48A0-B999-8629B42E4E23/layers/10801C0C-5E75-4492-A3EC-DAB0ECA3C56F)

**Comment by aufdenkampe** [comment]
For the "Catchments" List tab when viewing **_HUC-12-level model results_** for Water Quality, please mimic the following table, which has extra columns to the one shown above. The live mockup can be found in the [Catchments List Table](https://docs.google.com/spreadsheets/d/1OzuN-3SV3_CHrVDoHSRd7ukGMCvVrWpUnnMVik_ZpT8/edit#gid=1066341079) Google Sheet.

NOTE: 
* this table is _nearly identical to the HUC-12 list table_ referenced in #2726, except for the first few columns.
* this table mimics the AoI-level and HUC-12-level Sources tables_ referenced in #2724 & #2729 respectively.

<img width="812" alt="screen shot 2018-04-04 at 11 40 38 am" src="https://user-images.githubusercontent.com/5166036/38321412-3ca7afb6-37fd-11e8-9ead-e3555511a806.png">

**Comment by aufdenkampe** [comment]
I made a mistake by omitting three columns for "Mean Annual Concentration (discharge normalized) and also a column for Discharge, as we had discussed and decided under Topic #1b & 1c in [2018-02-12 Touching Base on Sub-Basin UI​](https://docs.google.com/document/d/1mG6m6wxxJjoSxK4SyYrNRaUTHZujSHYz5B7aXmCJWsQ/edit#bookmark=kix.e2poxslcum0z) Google Doc meeting notes (note that 1C is where we recorded our decisions, so if there is inconsistency between 1b & 1c, I'm going with 1C). 
My apologies for missing that we decided to exclude that for all the other tables, but not for the Catchments list table.

Remember that we want to visualize the Mean Annual Concentrations for the stream segments corresponding to each of these catchments.

This is what the Catchments List Table should look like:
<img width="1008" alt="screen shot 2018-04-05 at 10 59 14 am" src="https://user-images.githubusercontent.com/5166036/38377615-f1465ba2-38c0-11e8-94ec-7bb85a7be16a.png">

I need to get some input on the units for discharge.
